site/content/docs/user/quick-start.md

```diff
@@ -128,7 +128,7 @@ This allows a workflow like:
 ```
 docker build -t my-custom-image:unique-tag ./my-image-dir
 kind load docker-image my-custom-image:unique-tag
-kubectl apply -f my-manifest-using-my-image:unique-tag
+kubectl apply -f my-manifest-using-docker.io/my-custom-image:unique-tag
```

Comment:
As long as this is happening I think we need to be a little more explicit about the loaded image being prefixed with docker.io/ if there is no registry.

TODO(bentheelder): reproduce this locally and look into avoiding the explicit prefix.

I'm also slightly surprised that this didn't work regardless of the stored image including docker.io, I thought kubernetes could map `image: foo` -> `image: docker.io/foo:latest` -> a specific image with a hash 😞

Reply:
happy to set up a meeting and replicate it together :)